test/test_non_overlap_module.py

```diff
@@ -0,0 +1,106 @@
+'''
```

Comment:
2. updated alert message
3. updated placeholder value
5. updated description and tooltip when hovering over description